### PR TITLE
Add br.gov.fazenda.receita.irpf2024

### PR DESCRIPTION
**Pull request by guihkx**
### Please confirm your submission meets all the criteria

Please describe your application briefly below.

IRPF is a Java app distributed by the Special Department of Federal Revenue of Brazil, to assist Brazilian citizens in filing their annual tax return. They release one major version every year, and this is the 2024 edition.

- [x] I have read the [App Requirements](https://docs.flathub.org/docs/for-app-authors/requirements) and [App Maintenance](https://docs.flathub.org/docs/for-app-authors/maintanance) pages.
- [x] My pull request follows the instructions at [App Submission](https://docs.flathub.org/docs/for-app-authors/submission).
- [x] I have [built](https://docs.flathub.org/docs/for-app-authors/submission/#before-submission) and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

### CODE REVIEW

**Comment by guihkx** [comment]
bot, build br.gov.fazenda.receita.irpf2024

**Comment by bbhtt** [comment]
Why does each major version need its own Flatpak?

File: br.gov.fazenda.receita.irpf2024.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+  - name: irpf2024
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 -t /app/bin apply_extra
+      - install -Dm755 -t /app/bin irpf2024
+      - install -Dm644 -t /app/share/applications $FLATPAK_ID.desktop
+      - install -Dm644 icon-64.png /app/share/icons/hicolor/64x64/apps/$FLATPAK_ID.png
+      - install -Dm644 icon-128.png /app/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png
+      - install -Dm644 icon-256.png /app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
```
Comment (bbhtt) [inline br.gov.fazenda.receita.irpf2024.yaml:44]: Just do the 256px png, you can delete the rest.

**Comment by hfiguiere** [comment]
> Why does each major version need its own Flatpak?

This was already discussed and agreed

People need to be able to install the one for the previous year.

**Comment by guihkx** [comment]
bot, build br.gov.fazenda.receita.irpf2024

**Comment by farribeiro** [comment]
bot, build

**Comment by farribeiro** [comment]
bot, build br.gov.fazenda.receita.irpf2024

**Comment by farribeiro** [comment]
@guihkx Don't you want to wait for the merge to then apply those updates to the app?

**Comment by guihkx** [comment]
Sure, but I don't think there's any harm in keeping them updated here, right now...

**Comment by farribeiro** [comment]
So you are using Extra-data

**Pull request closed without merging**